src/events/azure_sdk.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * @fileoverview Handlers for the azure-sdk js library instrumentation.
+ */
+const utils = require('../utils.js');
+const tracer = require('../tracer.js');
+const eventInterface = require('../event.js');
+const moduleUtils = require('./module_utils.js');
+
+/**
+ * Wraps the BlockBlobClient upload method.
+ * @param {Function} wrappedFunction The function to wrap
+ * @returns {Function} The wrapped function
+ */
+function blobUploadWrapper(wrappedFunction) {
+    return function internalUploadWrapper(content, size) {
+        const { accountName, containerName } = this;
+        const { slsEvent: uploadEvent, startTime } = eventInterface.initializeEvent(
+            'blob_storage',
+            containerName,
+            'upload',
+            'azure-sdk'
+        );
+        eventInterface.addToMetadata(uploadEvent, {
+            'azure.blob.account_name': accountName,
+            'azure.blob.container_name': containerName,
+            'azure.blob.content_size': size,
+        }, { 'azure.blob.content': content });
```

Comment:
never upload the content (not even for metadata)

Reply:
CosmosDB, not storage :)